src/sort-task-array.js

```diff
@@ -0,0 +1,16 @@
+const sortTaskList = (list) => {
+  let temp;
+  for (let i = 0; i < list.length; i += 1) {
```

Comment:
Here would be a great opportunity to use the .map() method to iterate through this array! :smile:  Not only does it save you from writing complicated for loops, but it also creates a new array, instead of touching the original one, which is safer and a good practice overall.